docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst

```diff
@@ -54,13 +54,7 @@ Because Ansible is a flexible tool, there are a number of ways to specify connec
    ansible_become_method=enable
    ansible_network_os=eos
    ansible_user=my_eos_user
-   ansible_password= !vault |
```

Comment:
Why is vault password example removed? Storing password in plain text is not recommended.

Reply:
ini-style inventory does not support Ansible Vault. YAML inventory only support Ansible Vault.

Reply:
Ah I missed that.